.github/workflows/push.yml

```diff
@@ -198,12 +198,114 @@ jobs:
         env:
           VERSION: ${{ needs.set-version.outputs.version }}
 
+  terraform-testing-deploy:
+    name: 'Deploy Terraform to Testing'
+    needs:
+      - set-version
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    defaults:
+      run:
+        working-directory: provisioning/terraform
+    env:
+      TF_VERSION: 1.1.4
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: ${{ env.TF_VERSION }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::813746015128:role/kbc-github-actions-admin-role
+          aws-region: eu-central-1
+      - name: Terraform Init
+        run: terraform init -backend-config=backend_testing.conf -input=false
+      - name: Terraform Validate
+        id: validate
+        run: terraform validate
+      - name: Terraform Plan
+        run: terraform plan -var-file="testing.tfvars" -input=false
+      - name: Terraform Apply
+        run: terraform apply -auto-approve -var-file="testing.tfvars" -input=false
```

Comment:
Tak jsem to víceméně jen zkopíroval z https://github.com/keboola/keboola-assets-cdn/tree/main/.github/workflows ale `terraform apply` failuje na tom že se zjevně snaží všechno vytvořit znovu: https://github.com/keboola/keboola-as-code/actions/runs/3225914493/jobs/5278798548 Přitom backend je initnutý, step "Configure AWS Credentials" by měl použít roli co jsem vytvořil a následně `Terraform Init` by se měl napojit na ten backend a tam zjistit, že ty resoursy už existují, ne?

Reply:
Jdu mrknout

Reply:
Přijde mi jakoby to tam bylo vytvořeno předtím bokem ty resources, stáhnul jsem si ten state a ten `cli_dist_bucket` na kterém to padá tam vůbec nevidím. Možná si to tam vyslal z lokálu předtím bez remote state?

Reply:
🤔 Nó jo, poslal jsem to tam z lokálu (těma stejnýma commandama, vlastně). Myslel jsem právě že si v tom backendu v S3 zjistí co už v AWS existuje. Takhle to teda nefunguje, jo? 😄 Zkusím teda všechno smazat a nechat to nahodit jen z CI.

Reply:
normalne by to tak fungovalo, jenom si nejsem jisty jestli to z toho lokalu teda pouzilo stejny backend, vypada to ze ne.

Reply:
Pokud jsi pro nahození použil local backend a ted to chceš přepnout na remote v rámci jednoho aws accountu, tak bez migrace statu nepůjde. Ten state je třeba do S3 přenést, jde to manuálně (upload přes UI) nebo pře cli. Ale smazání pomůže taky, pokud to není problém udělat :)

Reply:
Jo to bude ono, už vím co se stalo. Až dodatečně jsem tam přidával tohle. 😅  Chybělo mi to oproti tomu `keboola-assets`: 
https://github.com/keboola/keboola-as-code/blob/234d76bdc7caea3c572876cbd5a51e27a6ebe4e7/provisioning/terraform/backend.tf#L1-L5